og.api.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * @file
+ * Hooks provided by the Organic Groups module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Access\AccessResultInterface;
```

Comment:
We're using AccessResultAllowed later on:
```suggestion
use Drupal\Core\Access\AccessResultAllowed;
use Drupal\Core\Access\AccessResultInterface;
```

But I 'd suggest using AccessResult:allowed()
```suggestion
use Drupal\Core\Access\AccessResult;
use Drupal\Core\Access\AccessResultInterface;
```